### ISSUE TITLE
Percy build fails on dark mode filter interaction

### ISSUE THREAD

**Issue by blms** [Princeton-CDH/geniza#727]
```
File "/home/runner/work/geniza/geniza/geniza/common/management/commands/visual_tests.py", line 77, in take_snapshots
  browser.find_element_by_css_selector(
...
raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
(Session info: headless chrome=98.0.4758.102)
```

**Comment by richmanrachel** [comment]
@blms - is this still a problem?

**Comment by blms** [comment]
Good catch @richmanrachel, looks like it was fixed in 70e6350760bd60ab96341f3595004a5153db3257! Closing.
